rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/vendor/VendorController.java

```diff
@@ -274,6 +274,56 @@ public ResponseEntity<?> exportVendor(HttpServletResponse response) throws TExce
         return ResponseEntity.ok(HttpStatus.OK);
     }
 
+    @PreAuthorize("hasAuthority('WRITE')")
+    @Operation(
+            summary = "Merge two vendors.",
+            responses = {
+                    @ApiResponse(responseCode = "200", description = "Merge successful.",
+                            content = {
+                                    @Content(mediaType = "application/json",
+                                            examples = @ExampleObject(
+                                                    value = "{\"message\": \"Merge successful.\"}"
+                                            ))
+                            }),
+                    @ApiResponse(responseCode = "400", description = "Merge failed.",
+                            content = {
+                                    @Content(mediaType = "application/json",
+                                            examples = @ExampleObject(
+                                                    value = "{\"message\": \"Merge failed.\"}"
+                                            ))
+                            }),
+                    @ApiResponse(responseCode = "409", description = "Merge failed because vendor is used.",
+                            content = {
+                                    @Content(mediaType = "application/json",
+                                            examples = @ExampleObject(
+                                                    value = "{\"message\": \"Merge failed because vendor is used.\"}"
+                                            ))
+                            }),
```

Comment:
`409` conflict is not returned by this method as far as I can see. Instead there is a `500` undocumented with message `Internal server error while merging the vendors`.

Maybe you meant with the `RequestStatus.IN_USE` scenario which will currently throw `400 BAD REQUEST` due to `HttpMessageNotReadableException`